tests/transactional/install_updates.pm

```diff
@@ -45,9 +45,16 @@ sub run {
         assert_script_run('journalctl --sync --flush --rotate --vacuum-time=1second');
         assert_script_run('rm /tmp/journal_before');
     }
+
+    # First we update the system
+    my $ret = trup_call 'up', timeout => 1800, proceed_on_failure => 1;
+    soft_fail_rt_scriptlet if ($ret != 0);
+    process_reboot(trigger => 1);
```

Comment:
Maybe you can just use `fully_patch_system` function from `utils.pm`.

Reply:
Good idea, trying it out now.